src/core/stdc/config.d

```diff
@@ -150,3 +155,15 @@ else version( SDC )
 }
 
 static assert(is(c_long_double), "c_long_double needs to be declared for this platform/architecture.");
+
+version (Darwin)
+{
+    alias cpp_size_t = cpp_ulong;
```

Comment:
What happens if we do the other way around, i.e. `alias size_t = cpp_ulong` in `object.d`? This would be to avoid having to use `cpp_size_t` instead of `size_t`.